vmm/src/lib.rs

```diff
@@ -920,6 +918,17 @@ impl Vmm {
                             .expect("Failed to register vcpu signal handler");
                         }
 
+                        // Load seccomp filters for this vCPU thread.
+                        // Execution panics if filters cannot be loaded, use --seccomp-level=0 if skipping filters
+                        // altogether is the desired behaviour.
+                        match default_syscalls::set_seccomp_level(seccomp_level) {
```

Comment:
The commit message is in contradiction with what happens here. From what I can see, the filters need to be loaded on each thread after creation, so vcpus won't inherit them from the vmm thread,

Reply:
Also, this can be rewritten with an `if let` instead of s `match`